src/sentry/api/helpers/group_index/update.py

```diff
@@ -264,305 +254,18 @@ def update_groups(
             project_lookup=project_lookup,
         )
     if status in ("resolved", "resolvedInNextRelease"):
-        res_status = None
```

Comment:
All code in here has been moved into the function with minimal (I will point it out if I do) or no code changes.